op-node/rollup/derive/channel_bank.go

```diff
@@ -89,7 +89,7 @@ func (cb *ChannelBank) IngestFrame(f Frame) {
 			cb.metrics.RecordHeadChannelOpened()
 		}
 		// create new channel if it doesn't exist yet
-		currentCh = NewChannel(f.ID, origin)
+		currentCh = NewChannel(f.ID, origin, false)
```

Comment:
I'm surprised this is hard coded to false - is this part of it not being wired in yet? I'm just slightly nervous that this would be easy to miss updating later. Maybe its worth adding a `TODO` that references the issue so we get pinged if the issue is closed without resolving this?

Reply:
It's hard coded to `false` on purpose because the `ChannelBank` is only used pre-Holocene, and the `ChannelStage` post-Holocene. So the `ChannelStage` also hard-codes `true`.